ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
```

Comment:
if we continue to have two month breaks between releases, I think it's important to tell people these features are coming. we keep getting the same issues filed over and over (video on mobile, for example); folks don't search issues before filing. so I think it's good to have a ROADMAP that's very up front about what we intend to fix and in which release we intend to ship.

> personally i think the milestone tagging stuff is a bit too much process and not that important, but you can

if releases were more frequent (such as a patch release after verified fix), I'd say this would be unnecessary, yes.